src/config.example.ts

```diff
@@ -2,7 +2,7 @@
 
 import { AuthConfig } from "./access";
 
-export default {
+const config: AuthConfig = {
```

Comment:
this is unrelated, but it means that the Typescript compiler will actually check that `permissions` is in the right format.